parser/src/cfg/defcfg.rs

```diff
@@ -77,6 +79,8 @@ impl Default for CfgOptions {
                 target_os = "unknown"
             ))]
             windows_interception_mouse_hwid: None,
+            #[cfg(target_os = "macos")]
```

Comment:
```suggestion
            #[cfg(any(target_os = "macos", target_os = "unknown"))]
```
similar to above (https://github.com/jtroo/kanata/pull/652/files#r1419935217)